### ISSUE TITLE
JupyterLab installed for Python 3.9 instead of 3.10 in Universal image

### ISSUE THREAD

**Issue by dmgardiner25** [devcontainers/images#125]
When trying to go through the [codespaces-getting-started-ml](https://github.com/github/codespaces-getting-started-ml) sample repo in JupyterLab, we are failing to import some packages. This is due to us installing `jupyterlab` with python 3.9 instead of 3.10. The same does not happen in VS Code.

Repro:
1. Create a codespace using the repo mentioned above
2. Connect to JupyterLab using `gh cs jupyter`
3. Attempt to run the first cell and notice that it fails to import `torch`

Notes:
- You can verify that this is the issue by running `pip list` to see that `jupyterlab` is missing from the list of python 3.10 packages and `/usr/local/python/3.9.14/bin/pip list` to see that it's installed under python 3.9`

cc. @jungaretti @samruddhikhandale

**Comment by jungaretti** [comment]
`pip`, `pip3`, `python`, and `python3` are all pointing to the correct installation of Python:

```sh
which pip && which pip3 && which python && which python3 && python --version
```
```sh
/home/codespace/.python/current/bin/pip
/home/codespace/.python/current/bin/pip3
/home/codespace/.python/current/bin/python
/home/codespace/.python/current/bin/python3
Python 3.10.7
```

Notice that the output of `pip list` does not include `jupyterlab`:

```sh
pip list | grep jupyter
```

(No output)

However, `/usr/local/python/3.9.14/bin/pip list | grep jupyter` does include `jupyterlab`:

```sh
/usr/local/python/3.9.14/bin/pip list | grep jupyter
```
```sh
jupyter_client       7.3.5
jupyter-core         4.11.1
jupyter-server       1.18.1
jupyterlab           3.4.7
jupyterlab-pygments  0.2.2
jupyterlab_server    2.15.1
```

I suspect this is a bug in the Python feature that installs JupyterLab to the "additional version" of Python that we're installing here: https://github.com/devcontainers/images/blob/4690ca1ee6b0ea5c3bef39b14e46c0c33d01824f/src/universal/.devcontainer/devcontainer.json#L27

**Comment by jungaretti** [comment]
We use the Python feature's `install_user_package` helper to install JupyterLab: https://github.com/devcontainers/features/blob/ceb4544cc38d4882a0294c79f4ef8123435cb027/src/python/install.sh#L316-L319

`install_user_package` installs packages to whatever Python installation is specified by `$INSTALL_PATH`. Both `install_from_source` and `install_using_oryx` overwrite this variable. When we use `install_python` to install the extra version of Python (3.9), we overwrite `$INSTALL_PATH` to that installation.

When we install JupyterLab at the end of the script, it uses 3.9's `$INSTALL_PATH` instead of 3.10.

**Comment by jungaretti** [comment]
This is fixed in version `2.0.16` of the Universal image.
